hexa/pipelines/urls.py

```diff
@@ -7,6 +7,7 @@
 app_name = "pipelines"
 
 urlpatterns = [
+    path("run", views.run_pipeline, name="run"),
```

Comment:
I think that the URL should contain the pipeline ID, and optionally the version.

It's easier for the webhook use case, especially for the use case that triggered the development of this feature (SMS -> Twilio number > webhook), that way the pipeline ID does not have to be sent in the SMS payload.